libs/chainscript/src/test/mod.rs

```diff
@@ -0,0 +1,56 @@
+use crate::*;
+
+// Test the interpreter on all 4-byte combinations of non-trivial opcodes.
+#[test]
+//#[ignore = "This test is too expensive to run by default"]
```

Comment:
Did you mean to comment the ignore?

Reply:
Not sure. It runs pretty quickly in release mode, but under debug, I will have to measure.

Reply:
Running this test with debug build takes 147 seconds on my laptop.

Reply:
What do we aim for in these tests? Are tests that run a long-ish time under debug acceptable or do we want to disable them by default (make opt-in)?

Reply:
147s is okay for me but is probably approaching the limit of "I'm getting bored waiting for this"-ness so maybe making them opt-in is a good idea. If someone touches the script stuff then they know to run them anyway so we're probably safe.

Reply:
In release mode, it is about 5s on my laptop btw